content/applications/inventory_and_mrp/quality/quality_reporting/quality_alerts_report.rst

```diff
@@ -0,0 +1,210 @@
+=====================
+Quality alerts report
+=====================
+
+In Odoo's *Quality* app, the quality alerts report displays key statistics about existing alerts.
```

Comment:
```suggestion
In Odoo's *Quality* app, the *Quality Alerts* report displays key statistics about existing alerts.
```
suggesting the use of italics here because it is the name of the reporting page that appears on the UI, but since it's not "presently" on the screen at this point, *italics* would be used.